src/common/pre-packaging-helper.sh

```diff
@@ -34,6 +34,27 @@ setup_maven(){
   export PATH=$M2_HOME/bin:$PATH
 }
 
+
```

Comment:
It might make more sense to split java and node, that way you only get what you need. Or we just come up with a different name. pre-packaging doesn't mean much to me.
I prefer the split scripts. one for java and one for node.